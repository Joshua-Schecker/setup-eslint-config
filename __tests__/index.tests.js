const { createConfig } = require('../');

test('createConfig', async () => {
  const setupConfig = {
    prompts: [],
    createDependencyList: () => ['eslint'],
    packageInfo: { devDependencies: { eslint: '^42' } },
  };

  expect(await createConfig(setupConfig)).toMatchSnapshot();
});
