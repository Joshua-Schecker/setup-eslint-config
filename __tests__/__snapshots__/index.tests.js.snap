// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createConfig 1`] = `
Object {
  "babel": false,
  "dependencies": Array [
    "eslint@^42",
  ],
  "flowtype": false,
  "jest": false,
  "prettier": false,
  "react": false,
  "typescript": false,
  "vue": false,
  "yarn": true,
}
`;
